src/components/molecules/TableGrid/index.ts

```diff
@@ -0,0 +1 @@
+export { TableGrid } from "./TableGrid";
```

Comment:
Another file that should probably be deleted `src/components/molecules/TablesUserList/TablesUserList.scss` if GitHub is correctly displaying it as existing but empty

Reply:
Already deleted.